frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/VerifyCodeHandler.java

```diff
@@ -158,29 +158,8 @@ public APIGatewayProxyResponseEvent handleRequestWithUserContext(
                 return generateApiGatewayProxyErrorResponse(400, ErrorResponse.ERROR_1049);
             }
 
-            UserProfile userProfile = null;
-
-            if (userProfileMaybe.isPresent()) {
-                userProfile = userProfileMaybe.get();
-            }
-
-            if (journeyType == JourneyType.REAUTHENTICATION
-                    && configurationService.isAuthenticationAttemptsServiceEnabled()) {
-                var countsByJourney =
-                        authenticationAttemptsService.getCountsByJourney(
-                                userProfile.getSubjectID(), JourneyType.REAUTHENTICATION);
-
-                var countTypesWhereBlocked =
-                        ReauthAuthenticationAttemptsHelper.countTypesWhereUserIsBlockedForReauth(
-                                countsByJourney, configurationService);
-
-                if (!countTypesWhereBlocked.isEmpty()) {
-                    LOG.info(
-                            "Re-authentication locked due to {} counts exceeded.",
-                            countTypesWhereBlocked);
-                    return generateApiGatewayProxyErrorResponse(400, ErrorResponse.ERROR_1057);
-                }
-            }
+            if (checkErrorCountsForReauth(journeyType, userProfileMaybe))
```

Comment:
I'm not suggesting to fix as part of this PR, but maybe for a follow up if we get to it, a more descriptive name for this might be "anyRetryCountsForReauthExceededOnReauthJourney" or something that indicates it's returning a bool. Also think might read better for me if the "if reauth journey" is done at this level, and it's clear we're only checking in the case of a reauth journey